scripts/dist-select.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('node:fs');
+const { ArgumentParser } = require('argparse');
+const { switchVersion, loadModule } = require('./utils');
+const Vue = loadModule('vue');
+
+if (!fs.existsSync('dist')) {
+  return;
+}
+
+const parser = new ArgumentParser();
+parser.add_argument('--force-version', { type: 'str' });
+
+const args = parser.parse_args();
+const { force_version } = args;
+
+if (force_version !== 'undefined') {
+  if (force_version === '3') {
+    switchVersion(3);
+  } else {
+    switchVersion(2);
+  }
+
+  return;
+}
+
+if (!Vue || typeof Vue.version !== 'string') {
+  console.warn(
+    '[vue3-sketch-ruler] Vue is not found. Please run "npm install vue" to install.'
```

Comment:
this will appear in the consumer?

Reply:
consumer as the one that uses `yarn|npm|pnpm install @rotki/ui-library@x` if they don't see and this is only for the developer building the package then we should change this `npm` -> `pnpm`

Reply:
effect of copy pastaa 😛